flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCTableSourceSinkFactoryTest.java

```diff
@@ -182,6 +184,12 @@ public void testJDBCWithFilter() {
 		assertEquals(projectedFields.get("aaa"), DataTypes.INT());
 		assertNull(projectedFields.get("bbb"));
 		assertEquals(projectedFields.get("ccc"), DataTypes.DOUBLE());
+
+		// test jdbc table source description
+		List<String> fieldNames = ((RowType) actual.getProducedDataType().getLogicalType()).getFieldNames();
+		String expectedSourceDescription = actual.getClass().getSimpleName()
+			+ "(" + String.join(", ", fieldNames.stream().toArray(String[]::new)) + ")";
```

Comment:
I think we should add an integreation test to verify the JDBC source can work when project is pushed down.

Reply:
@wuchong Good idea. I have updated.